models/gamma_sup_generator.h

```diff
@@ -61,6 +61,9 @@ gamma_shape
 n_proc
```

Comment:
Should these also be double backticked? Depends also on the style in the `include`d description.

Reply:
I saw this also in many later files. Do not fix this in this here though, we need a general approach to this. Just fixing this in all files could easily be its own PR.